docs/view_config.rst

```diff
@@ -40,32 +40,86 @@ json (e.g. `{"viewconf": myViewConfig}`):
          -d '{"viewconf": {"editable": true, "zoomFixed": false, "trackSourceServers": ["/api/v2", "http://higlass.io/api/v1"], "exportViewUrl": "/api/v1/viewconfs/", "views": [{"tracks": {"top": [], "left": [], "center": [], "right": [], "bottom": []}, "initialXDomain": [243883495.14563107, 2956116504.854369], "initialYDomain": [804660194.1747572, 2395339805.825243], "layout": {"w": 12, "h": 12, "x": 0, "y": 0, "i": "EwiSznw8ST2HF3CjHx-tCg", "moved": false, "static": false}, "uid": "EwiSznw8ST2HF3CjHx-tCg"}], "zoomLocks": {"locksByViewUid": {}, "locksDict": {}}, "locationLocks": {"locksByViewUid": {}, "locksDict": {}}, "valueScaleLocks": {"locksByViewUid": {}, "locksDict": {}}}}' http://localhost:8989/api/v1/viewconfs/
 
 Viewconf Structure
--------------------
+==================
+
+The viewconf defines which data is shown by HiGlass and how it is arranged. It
+is organized into views, tracks and other metadata. This section will focus on
+the root-level metadata. The sections below will delve into views and tracks.
+
+.. code-block:: javascript
+
+  editable: [true | false]
+
+Sepcifies whether this viewconf will have a view header.
+
+.. code-block:: javascript
+
+    {
+      "trackSourceServers": [
+        "http://higlass.io/api/v1",
+        "http://localhost:8989/api/v1"
+      ],
+    }
+
+Where should the list of available tracks be pulled from? The field
+`trackSourceServers` at the root level of the viewconf tells higlass where it
+can find tracks to load. If you have a local instance running, then
+`http://localhost:8989/api/v1` should be included. Our public instance at
+`http://higlass.io` also provides access to a number of public datasets.
+
+
+.. code-block:: javascript
+
+    {
+        "exportViewUrl": "/api/v1/viewconfs",
+    }
+
+The ``exportViewUrl`` field in the viewconf specifies which server should be used
+to store exported viewconfs. This server stores exported viewconfs in its 
+database, assigns them a uid, and makes them accessible through its API at
+``/api/v1/viewconfs/uid/``. 
```

Comment:
- This seems weird: if it's specifying a server, then there should be a host in the URL? Maybe explain it differently...
- Formatting: single backticks were sufficient in the last paragraph?

Reply:
You're right. It's not a server. It's an API endpoint.

Reply:
I'll change it.